.github/workflows/ci-actions.yml

```diff
@@ -0,0 +1,30 @@
+name: Jakarta Contexts and Dependency Injection CI
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    name: "Build CDI - JDK ${{matrix.java.name}}"
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        java:
+          - {
+            name: "11",
+            java-version: 11,
+          }
```

Comment:
You can do just

```yaml
jobs:
  build:
    strategy:
      fail-fast: false
      matrix:
        java:
        - 11
```

and later

```yaml
jobs:
  build:
    steps:
    - name: Set up JDK ${{ matrix.java }}
      uses: actions/setup-java@v2
      with:
        distribution: adopt
        java-version: ${{ matrix.java }}
```

(Ignore the `setup-java@v2` difference, I'm copying from a project where I use that, as it allows to choose the `distribution`.)

Reply:
But what you have now is of course also fine.

Reply:
Ok, I have copied this from a setup that I was using elsewhere. Don't recall exactly why I did it this way but I think I wasn't able to reference the version from job name (which might have been a bug back then)?
Can change it to what you are proposing.

Reply:
I don't really mind :-)